app/screens/sso/sso_with_redirect_url.tsx

```diff
@@ -63,7 +63,16 @@ function SSOWithRedirectURL({
                 defaultMessage: 'The link failed to open. Please try again.',
             }),
         );
-        tryOpenURL(url, onError);
+        try {
+            const result = await WebBrowser.openAuthSessionAsync(url, redirectUrl);
```

Comment:
I thought this was the other method

Reply:
I was testing with both the methods because of the discussed iOS issue. I have committed this to let our team to test if this solution (with any method) fixes the VPN tunnel issue or not.